tests/unit/dojson/test_dojson_hep.py

```diff
@@ -1609,25 +1614,42 @@ def test_mashed_publication_info_from_773():
         '  <subfield code="2">978924</subfield>'
         '  <subfield code="0">1408366</subfield>'
         '</datafield>'
-    ) # record/820763/export/xme
+    )  # record/820763/export/xme
 
-    expected = {
-        'journal_title': 'IAU Symp.',
-        'cnum': 'C08-06-09',
-        'journal_volume': '354',
-        'year': 2008,
-        'artid': '45',
-        'page_start': '45',
-        'journal_record': {
-            '$ref': 'http://localhost:5000/api/journals/1212883',
-        },
-        'parent_record': {
-            '$ref': 'http://localhost:5000/api/literature/1408366',
-        },
-        'conference_record': {
-            '$ref': 'http://localhost:5000/api/conferences/978924',
+    expected = [
+        {
+            'journal_title': 'IAU Symp.',
+            'cnum': 'C08-06-09',
+            'journal_volume': '354',
+            'year': 2008,
+            'artid': '45',
+            'page_start': '45',
+            'journal_record': {
+                '$ref': 'http://localhost:5000/api/journals/1212883',
+            },
+            'parent_record': {
+                '$ref': 'http://localhost:5000/api/literature/1408366',
+            },
+            'conference_record': {
+                '$ref': 'http://localhost:5000/api/conferences/978924',
+            },
         },
-    }
+    ]
     result = hep.do(create_record(snippet))
 
-    assert expected == result['publication_info'][0]
+    assert validate(result['publication_info'], subschema) is None
+    assert expected == result['publication_info']
+
+    expected = [
+        {
+            'c': ['45'],
+            'p': 'IAU Symp.',
+            '0': 1408366,
```

Comment:
I don't think we should be producing guys like these, because they come from the `xme` format, while here we are producing the `xm` format. This is a common problem in the MARCXML -> JSON rules as they are currently written.

CC: @kaplun